server/service/osquery.go

```diff
@@ -1013,6 +1013,10 @@ func (svc *Service) SubmitDistributedQueryResults(
 			logging.WithErr(ctx, err)
 		}
 
+		if err := svc.processScriptsForNewlyFailingPolicies(ctx, host.ID, host.TeamID, host.Platform, host.OrbitNodeKey, host.ScriptsEnabled, policyResults); err != nil {
```

Comment:
Load should be pretty comparable, and we should be able to load test in a similar fashion (albeit with no network load concerns since scripts are KB rather than MB or GB).